docs/monetization/Entitlements.md

```diff
@@ -0,0 +1,184 @@
+## Entitlement Resource
+
+Entitlements in Discord represent that a user or guild has access to a premium offering in your application.
+
+### Entitlement Object
+
+###### Entitlement Structure
+
+| Field          | Type              | Description                                                                                 |
+|----------------|-------------------|---------------------------------------------------------------------------------------------|
+| id             | snowflake         | ID of the entitlement                                                                       |
+| sku_id         | snowflake         | ID of the SKU                                                                               |
+| user_id?       | snowflake         | ID of the user that is granted access to the entitlement's sku                              |
+| guild_id?      | snowflake         | ID of the guild that is granted access to the entitlement's sku                             |
+| application_id | snowflake         | ID of the parent application                                                                |
+| type           | integer           | [Type of entitlement](#DOCS_MONETIZATION_ENTITLEMENTS/entitlement-object-entitlement-types) |
+| consumed       | boolean           | Not applicable for App Subscriptions. Subscriptions are not consumed and will be `false`    |
```

Comment:
```suggestion
```

if it doesn't apply for us, it doesn't need to be documented? (at least that's the normal behavior)